src/features/wallet/send/SendView.tsx

```diff
@@ -47,14 +39,16 @@ import {
 import { getAddress } from '../../../utils/secureAccount'
 import Text from '../../../components/Text'
 import useSubmitTxn from '../../../hooks/useSubmitTxn'
-import { ensLookup } from '../../../utils/explorerClient'
-import { decimalSeparator, groupSeparator, locale } from '../../../utils/i18n'
+import { decimalSeparator, groupSeparator } from '../../../utils/i18n'
 import { useAppDispatch } from '../../../store/store'
 import {
   fetchCurrentOraclePrice,
   fetchPredictedOraclePrice,
 } from '../../../store/helium/heliumDataSlice'
-import { AppLink, AppLinkType } from '../../../providers/appLinkTypes'
+import {
+  AppLink,
+  AppLinkCategoryType as AppLinkType,
```

Comment:
I might be missing something with TypeScript, but it doesn't look like `appLinkTypes.ts` exports an `AppLinkType`? My editor also complained about this, but seems like `AppLinkCategoryType` is what we want so I just imported that under a different name.

Reply:
Yeah there was a bug in the code. You are correct. I fixed it in one of my open PRs too, but this will probably merge first so ill just resolve the conflict.